lib/versioning/npm/index.ts

```diff
@@ -26,6 +26,16 @@ const isSingleVersion = (constraint: string): string =>
   isVersion(constraint) ||
   (constraint.startsWith('=') && isVersion(constraint.substring(1).trim()));
 
+function increment(version: string): string | null {
+  const match = version.match(
```

Comment:
Let's use the `semver` library instead of trying to parse ourselves. I think `semver.inc(version, 'patch')` might satisfy what we need.